timeseries_storage/blueflood/blueflood.go

```diff
@@ -269,11 +255,13 @@ func (b *Blueflood) FetchSingleTimeseries(request api.FetchTimeseriesRequest) (a
 			return nil
 		}
 		fullResolutionParsedResult, rawResult, err := b.fetch(request, fullResolutionQueryURL)
-		rawResults = append(rawResults, rawResult)
 		if err != nil {
 			log.Infof("FULL resolution data errored while parsing result: %s", err.Error())
 			return nil
 		}
+		if request.UserSpecifiableConfig.IncludeRawData {
```

Comment:
raw values are now stored as evaluation notes, instead of affecting return value